bindings/py/packaging/setup.py

```diff
@@ -238,7 +238,7 @@ def generateExtensions():
     #     https://opensourceforu.com/2010/OS/extending-python-via-shared-libraries
     #     https://docs.python.org/3/library/ctypes.html
     #     https://docs.python.org/2/library/imp.html
-    name="nupic.bindings",
+    name="htm.core",
```

Comment:
I don't think this is the repo name.  Its the name given to the package but yes, htm.core is a better name.